src/components/input-formik.js

```diff
@@ -0,0 +1,41 @@
+import { memo } from "react";
+import PropTypes from "prop-types";
+import { styled } from "@linaria/react";
+import { Label } from "./label";
+
+const InputFormik = ({ type = "text", label, name, value, onChange, id }) => {
```

Comment:
we already have an input component, why U create new

Reply:
fixed